lib/configuration/configuration.ts

```diff
@@ -53,33 +60,35 @@ export const getConfiguration = async (): Promise<void> => {
     const res = await axiosInstance.get<Configuration[]>(
       `${config.configurationApiUrlV1}/api/v1/configuration?types=MMH`,
     );
-    res.data.forEach(({ type, featureToggles }) => {
-      const toggles = featureToggleStore.getValue();
-      featureToggleStore.next({
-        ...toggles,
+    res.data.forEach(({ type, featureToggles, configuration }) => {
+      const configs = configurationStore.getValue();
+      configurationStore.next({
+        ...configs,
         [type]: {
           ...featureToggles,
+          ...configuration,
         },
       });
     });
     window.localStorage.setItem(
       "features",
-      JSON.stringify(featureToggleStore.getValue()),
+      JSON.stringify(configurationStore.getValue()),
     );
   } catch (e) {
     // TODO add logging for failed configuration
   }
 };
 
+export const hasAppConfig = (path: AppConfigPaths): string => {
```

Comment:
I think change the name of this method. the verbs `has` and `is` in methods indicate a boolean return type, where this is not doing that.